test_server.py

```diff
@@ -36,29 +40,100 @@
                     help='Run on HPC config')
 args = parser.parse_args()
 
-# Add YOLOv5 directory to the system path to import custom functions
-yolov5_path = "./"  # Adjust the path as necessary
-sys.path.append(yolov5_path)
+MODEL_DOWNLOAD_FOLDER = "./artifacts"
+OPENVINO_MODEL_FOLDER = "./best_openvino_model" # WARNING: THIS FOLDER MUST HAVE THIS NAME
+URI = "http://ec2-3-21-53-196.us-east-2.compute.amazonaws.com:5000/"
 
-# Import custom detection functions from the YOLOv5 implementation
-from detect import run, load_model
+# Ensure download folders exist
+# MLFlow will not create the folder for us
+os.makedirs(MODEL_DOWNLOAD_FOLDER, exist_ok=True)
+os.makedirs(OPENVINO_MODEL_FOLDER, exist_ok=True)
+
+mlflow.set_tracking_uri(URI)
+
+# Attempt to grab the latest run ID
+print("Getting latest run ID...")
+download_artifacts = True
+try:
+    RUNHISTORY = mlflow.search_runs(search_all_experiments=True)
+    RUNID = RUNHISTORY['run_id'][0]
+
+    print(f"\tDownloading: {RUNID}")
+except Exception:
+    download_artifacts = False
+    print("\t ERROR: Failed to get the latest run...")
+    print("\t Will attempt to use the last used model.")
+
+print()
 
 # YOLO model parameters
+weights = OPENVINO_MODEL_FOLDER
 use_openvino = True  # Whether to use OpenVINO for inference
```

Comment:
If you're ditching the conditional use of openvino, might as well get ride of the boolean as well